test/index.ts

```diff
@@ -1,7 +1,13 @@
 import * as assert from 'assert';
+import * as nock from 'nock';
 import * as r from 'request';
+
 import {teenyRequest} from '../src';
 
+nock('http://www.example.com').persist().get('/').reply(202, 'ok', {
```

Comment:
Persist is bad. Instead, please create a function that returns the scope, then check `scope.done()` in the function.  This helps a.) ensure the requested nock was actually executed and b.) ensure it was only executed the # of times you expect it to be :)

Reply:
fixed :)